docs/acceptance-criteria/uacs-rfc-19-random-beacon-v2.adoc

```diff
@@ -0,0 +1,607 @@
+  valid, then: +
+  - the group that leaked the signature should get terminated (in the same tx) +
+  - all members of the group should get slashed a fixed amount of T (in the same
+  tx) +
+  - notifier should receive fixed part of the total slashed amount (in the same
+  tx) +
+  - remaning part of the total slashed amount should be burned or allocated for
+  the rewards pool of the staking contract.
+  
+| Sortition pool - punishments
+| Member of the sortition pool can be punished with temporary exclusion from 
+  earning rewards as a result of: +
+  - inactivity or disqualification during off-chain DKG +
+  - misbehavior during the DKG submission.
+
+| Sortition pool - punishments
+| The operation excluding members from earning the rewards should be as cheap as
+  possible so that the member submitting the relay entry or the DKG result does
+  not have to pay any additional costs for that exclusion.
+
+| Sortition pool - rewards
+| Operators in the sortition pool can earn T for multiple activities: +
+  - for staying in the pool (given weekly, proportionally to stake) +
+  - for successful submission of the DKG result (fixed value) +
+  - for successful unlocking of the sortition pool if DKG timed out.
+
+| Sortition pool - locking
+| When the sortition pool is locked, no operator can enter the pool.
+
+| Sortition pool - locking
+| When the sortition pool is locked, no operator can leave the pool.
+
+| Sortition pool - locking
+| When the sortition pool is locked, no operator can update its state.
+
+| Sortition pool - locking
+| Once the sortition pool gets unlocked, operators can leave the pool.
+
+| Sortition pool - locking
+| Once the sortition pool gets unlocked, new operators can join the pool.
+
+| Sortition pool - locking
+| Once the sortition pool gets unlocked, operators can update their state.
+
+| Sortition pool - locking
+| Operator's staked amount can change regardless if the pool is at the moment in
+  the locked or in the unlocked state (e.g. operator can be slashed during the
+  lock as a result of relay entry timeout).
+
+// TODO: Once starting values for governable parameters are agreed upon,
+// add them to the UACs
+
+| Governable parameters
+| Every `N`-th relay request initiates creation of a new signing group.
+  This group creation frequency (`groupCreationFrequency`) should be a governable
+  parameter.
+
+| Governable parameters
+| The time since the creation of a new signing group after that group gets
+  expired (`groupLifetime`) should be a governable parameter and should be
+  initally set to `80640 blocks` (~2 weeks, assuming 15s block time).
+
+| Governable parameters
+| The length of the challenge period (`resultChallengePeriodLength`) should be a
+  governable parameter and should be initally set to `1440 blocks`.
+
+| Governable parameters
+| The slashing amount for submitting malicious DKG result
+  (`maliciousDkgResultSlashingAmount`) should be a governable parameter.
```

Comment:
Not sure if the slashing is for submitting the result or supporting. Asked in https://github.com/keep-network/keep-core/pull/2751/files#r761879103.